packages/@react-types/button/src/index.d.ts

```diff
@@ -45,13 +45,13 @@ export interface LinkButtonProps {
 
 interface AriaBaseButtonProps extends FocusableDOMProps, AriaLabelingProps {
   /** Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed. */
-  'aria-expanded'?: boolean,
+  'aria-expanded'?: boolean | 'true' | 'false',
```

Comment:
Adjusted some of the aria here so it falls in line with the HTMLAttributes definition